index.html

```diff
@@ -20,26 +16,227 @@
+                        Comment
+                    </button>
+
+                    <!--Modal second-->
+                    <div class="modal fade" id="modal_post2" tabindex="-1" role="dialog" aria-labelledby="contenedor" aria-hidden="true">
+                        <div class="modal-dialog">
+                            <div class="modal-content">
+                                <div class="modal-header">
+                                    <h4 class="modal-title" id="myModalLabel2">Post a Comment</h4>
+                                </div>
+
+                                <div class="modal-body">
+                                    <form name="name_form2">
+                                        Name <input type="text" class="form-control" maxlength="15" name="name_input2" value="" onfocus="javascript:this.value=''"/>
+                                    </form>
+                                    <div class="form-group">
+                                        Comment<textarea maxlength="150" class="form-control" id="comment_input2" onfocus="javascript:this.value=''">
+                                        </textarea>
+                                    </div>
+                                </div>
 
-    <div class="posts"></div>
+                                <div class="modal-footer">
+                                    <button type="button" class="btn btn-default" data-dismiss="modal" onclick="javascript:input[name=comment_input1].value=''">Close</button>
+                                    <button type="button" class="btn btn-primary" data-dismiss="modal" id="save_comment2">Post</button>
+                                </div>
+                            </div>
+                        </div>
+                    </div> <!--End Modal-->
 
+                    <p class="info2"></p>
+                </div>
+            </div> <!--En of .one_two_posts-->
 
-    <!-- Bootstrap core JavaScript
-    ================================================== -->
-    <!-- Placed at the end of the document so the pages load faster -->
-    <script src="js/libs/jquery-2.1.1.js"></script>
-    <script src="js/script.js"></script>
-    <script src="js/libs/bootstrap.js"></script>
+
+            <div class="row three_four_posts">
+                <div class="col-md-6 col-md-offset-3 upper_post">
+                    <h1 class="display_content title3" id = "third_post">
+                    </h1>
+                    <p class="posts3"></p>
+                    <h4 class="title_comment" id="title_cmmt3">Comments</h4>
+                    <div class="comments_user" id="comment_div3"></div>
+
+                    <button type="button" class="btn btn-primary show_modal" data-toggle="modal" data-target="#modal_post3" id="commentary_button3">
+                        Comment
+                    </button>
+
+                    <!--third modal-->
```

Comment:
Great work documenting your code. You can think of the comments in a code as the intentions or thoughts of the programmer in the moment, which can clarify things when someone else reads it.